portsyncd/linksync.cpp

```diff
@@ -200,17 +197,12 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
             return;
         }
 
+        /* Host interface is created */
         if (!g_init && g_portSet.find(key) != g_portSet.end())
         {
-            /* Bring up the front panel port as the first place*/
-            /* TODO: handle system retur code, if-block is just a warning suppressor */
-            if (system(("/sbin/ifup --force " + key).c_str()))
-                ;
             g_portSet.erase(key);
```

Comment:
line 53 indicates that the admin status is already in the db, which indicates that the host interface is already created. the g_portSet is only used to track the ports that don't have host interfaces created yet.
this part of the code is to handle portsyncd restart issues so that after portsyncd restarts, it could still finish all the uninitialized ports/VLANs.